hack/cluster-build.sh

```diff
@@ -56,6 +56,12 @@ if [[ $image_prefix_alt ]]; then
     done
 fi
 
+# Prefetch cdi images
+cdi_images=$(sed -ne "s/.*\(quay.io[^\s]*\)/\1/p" manifests/testing/cdi*)
+for arg in ${cdi_images}; do
+    container="${container} ${arg}"
+done
+
```

Comment:
Will this run for any lane?
The SR-IOV lane (kind base) does not use CDI at all per what I remember.

@ormergi , can you please comment on this?

Reply:
> Please include the reasoning for this change in the commit message.

Done: will push it later.

> And another question: Will this run with any cluster or only with kubevirtci?

The change adds cdi containers to the kubevirt containers list, so it works on clusters where the 
 kubevirt pre-fetch works. 

I am not sure where the cluster-build.sh is used and how clusters without kubevirtci are built. Anyone can comment on this?

Reply:
@brybacki @EdDev 
I think it will run on sriov lane as well, we better move this closer to the CDI deployment code 
https://github.com/kubevirt/kubevirt/blob/master/hack/cluster-deploy.sh#L35-L70

Reply:
move to cluster-deploy done, also tried to extract some commons, I hope that is good now

Reply:
@ormergi @EdDev Is it good now?